ui_testing/testcases/basic_tests/test03_orders.py

```diff
@@ -750,3 +750,65 @@ def test017_update_shipment_date(self, save):
                 ' + Assert {} (current_shipment_date) == {} (order_shipment-date)'.format(current_shipment_date,
                                                                                           order_shipment_date))
             self.assertEqual(current_shipment_date, order_shipment_date)
+
+    @parameterized.expand(['save_btn', 'cancel'])
+    def test020_update_material_type_with_test_unit(self, save):
+        """
+
+        New: Orders: Material type Approach: I can update the material type
+        filed with test units records successfully
+
+        LIMS-3844
+        LIMS-3844
+        :return:
+        """
+
+        self.order_page.get_orders_page()
+        self.order_page.click_create_order_button()
+        order_url = self.base_selenium.get_url()
+        self.base_selenium.LOGGER.info(' + order_url : {}'.format(order_url))
+        self.order_page.set_new_order()
+        self.order_page.set_contact()
+        self.order_page.set_material_type(
+            material_type='r')
+        self.order_page.set_article(
+            article='a')
+        self.order_page.set_test_unit(
+            test_unit='f')
+        self.order_page.save(save_btn='order:save_btn')
+        self.order_page.get_orders_page()
+        orders_result = self.orders_page.result_table()
+        self.order_page.get_random_x(row=orders_result[0])
```

Comment:
remove lines from 779:781

Reply:
done

Reply:
@0xIslamTaha  this opens the last record I created so how can I open it

Reply:
create a method which take the order no and open it is page. it is code can be look like 
```
        self.base_selenium.LOGGER.info(
            ' + orders_created_with_number : {}'.format(order_no_created))
        order_no_created = order_no_created.replace("'", '')

        # filter by the created order number and get the count
        orders_result = self.orders_page.search(order_no_created)
        self.base_selenium.LOGGER.info(
            ' + filter_by_order_no : {}'.format(order_no_created))
        orders_count = self.order_page.get_table_records()
        self.base_selenium.LOGGER.info(
            ' + count_of_the_created_orders : {}'.format(orders_count))

        # open the last created order to update its number and checking whether it will affect the rest of the orders or not
        self.order_page.get_random_x(row=orders_result[0])

```

Reply:
@0xIslamTaha  but the last one I created to open the last record why I do this method this method is different

Reply:
this code to compare the counts and my code just to open the last record I don't need to search or filter